### DISCUSSION TITLE
Nel codice si utilizza il typescript ?

### DISCUSSION THREAD

**Discussion by mbico** [lifeisfoo/react-guida-pratica#14] [Q&A]
Sto cercando un testo per iniziare react >= 18 con typescript. 
Non trovo info se si utilizza lo typescritp.
Grazie

**Comment by lifeisfoo** [comment]
Il libro è su React e solo su React.
Alla fine del volume viene mostrato come usare TypeScript con React ma non è l'argomento principale. 
Se conosci già TypeScript non dovresti aver problemi a seguire e riscrivere gli esesmpi in JavaScript.
Se non conosci nessuno dei due ti sconsiglio di cercare di impararli insieme.
Spero che questa risposta ti sia utile.

**Reply by mbico** [reply]
Grazie, ha un consiglio su un secondo libro su TypeScript ?

**Reply by lifeisfoo** [reply]
Purtroppo no, mi dispiace.
Ne esistono tanti ma finora non ne ho letto nessuno, quindi non posso aiutarti.
